app/forms/cannot_use_service_form.rb

```diff
@@ -0,0 +1,7 @@
+class CannotUseServiceForm
```

Comment:
is this necessary? The page is a static page with nothing that the user needs to complete. This feels like it is added just to appears the pattern in 
`app/controllers/flow/handler.rb` but without the need for a form existing.
As this is a static page could we create a static_pages controller and manage this page that way?
https://greg.molnar.io/blog/rails-tricks-issue-15/

Reply:
@willc-work I think this is necessary and would prefer to use the `app/controllers/flow/handler.rb` logic for the following reasons:
- we use a similar pattern for the `app/forms/how_to_aggregate_form.rb` form and screen
- currently let's us tap into question flow logic but if we used a static page (as per link) might be convoluted to get it to work with `app/controllers/flow/handler.rb`, `app/lib/steps/logic.rb` & `app/controllers/change_answers_controller.rb` logic
- if we ever need to add any additional buttons etc we can add to the form model.

Reply:
I am happy that this can be merged and we write a ticket to refactor this so we can merge this change and continue with the development of the feature, but lets keep an eye on how much work that refactor becomes. If it is a significant rework by the time we get to it then we may want to bear that in mind for future similar decisions.

Reply:
@kmahern happy to pair and attempt that on another refactor ticket. 🙂

Reply:
approving based on the creation of a ticket to look at refactoring this